### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by CMUgarte**
Clean up `etc/languages.tsv `file and standard update.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
-Patience Epps | | | Author, Editor
-Jane Hill | | | Author, Editor
-Patrick McConvell | | | Author, Editor
-Robert Forkel | @xrotwang | code | Other
-Christoph Rzymski | @chrzyki | patron | Other
-Johann-Mattis List | @lingulist | orthography profile | Other
+Claire Bowern | | | Author, Other
+Patience Epps | | | Author, Other
+Jane Hill | | | Author, Other
+Patrick McConvell | | | Author, Other
```
Comment (FredericBlum) [inline CONTRIBUTORS.md]: Only Author, no need to add Other

**Review by FredericBlum — CHANGES_REQUESTED** [review]
We have a problem. The raw data is automatically downloaded! Check the download command... so any change in the raw file is overwritten through a data update (which you should have run in the very first place). And the Glottolog Name is something that we can retrieve automatically through the Glottocode. 

We never ever change the raw data for datasets which we download. Please remember that. I am afraid that was a lot of work for you that you will have to revert now. 

Please

a) Revert all the changes to the raw file
b) Download the data
c) Restore the second column (!) of the etc/languages.csv file - but only the second column, not the other changes! To avoid further work. You can checkout the old file through the commit history and then just copy&paste the column, to save time.

I propose to do this once the other datasets are finished, to make sure that we identify any major problems in other datasets first.

**Review by LinguList — APPROVED** [review]
Thanks, this will boost the coverage for segmented languages :-)

**Comment by LinguList** [comment]
Ah, I was wondering, why `cldfbench download` was run on the data. Did this destroy anything? Ideally, it would have been consistent, if they did not change their data. We may quickly discuss how to proceed in such cases. If huntergatherer keeps evolving, we may want to account for recent updates -- even if not for now (where we have the orthoprofile fixed)...

**Comment by FredericBlum** [comment]
The problem are the manual changes in the etc/languages, 'fixing' the language names both there and in the raw data. If we now download the data again (which we should have the possibility to, I think, in case they update), all those changes lead to ID errors between the raw data and the etc/languages

**Comment by LinguList** [comment]
Ah, that is of course unfortunate.

**Comment by CMUgarte** [comment]
@FredericBlum I adopted the workflow you suggested and should now be fine. Most of the work on this dataset was the search for Glottocodes, so it's nice that this PR can contribute with that as well.

**Review by chrzyki — APPROVED** [review]

**Pull request merged**